package.json

```diff
@@ -93,7 +93,7 @@
     "postcss-csso": "^5.0.0",
     "sass": "^1.35.1",
     "snyk": "^1.667.0",
-    "uswds": "^2.12.0",
+    "uswds": "github:uswds/uswds#dw-external-link",
```

Comment:
Using an icon feature branch — this will need to be updated (and package-lock generated) after we merge that branch into develop